portfolio/src/main/webapp/script.js

```diff
@@ -16,8 +16,11 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = [
+    'Hi', 'Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!',
+    'Greeting',
+    'According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don\'t care what humans think is impossible.'
```

Comment:
Is this auto-formatted by the tool I recommended? I think the wrapped lines of this "greetings" variable should be 4 spaces indention, if you notice this in the original code. Feel free to correct me if I'm wrong

Reply:
This is what the auto-formatter did

Reply:
Looks like against style used internally, but there is no need to too picky on this, as long as the style is consistent across the whole codebase.